examples/gradle/dokka-customFormat-example/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ buildscript {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
minor/major: not sure that it's a good idea to add `mavenLocal` to examples which are used also by external users which want to look how `dokka` is configured.
The main issue is that users often just `copy-paste` everything what is in file and then they can leave `mavenLocal` there and later accidentally have issues because of it.

I think, it will be better to add `mavenLocal` in this case via script if possible

Reply:
Yep, thanks, I agree that adding `mavenLocal()` via script is more flexible